src/library/impl/meta_reader/signature.h

```diff
@@ -154,9 +154,14 @@ namespace xlang::meta::reader
         uint32_t index;
     };
 
+    struct GenericMethodIndex
```

Comment:
Let's change this to GenericMethodTypeIndex

Reply:
The philosophy is to use the name exactly as it appears in the ECMA-335 spec, or as close to it as is practical, to make it easier to cross-reference.

Reply:
ECMA-335 isn't very rigorous in deciding what to name this construct. I think the closest thing we have is for ELEMENT_TYPE_VAR and ELEMENT_TYPE_MVAR, which are described as "Generic parameter in a generic type definition" and "Generic parameter in a generic method definition".

Maybe GenericTypeParameter and GenericMethodParameter would be the clearest naming?

Reply:
I still think GenericMethodTypeIndex is the right choice. Arguably GenericMethodTypeParameterIndex would be better, but I think that's getting too long. Plus, leaving "parameter" off helps distinguish from the normal method parameters